pkg/es/config/config.go

```diff
@@ -301,17 +291,14 @@ func (c *Configuration) getConfigOptions(logger *zap.Logger) ([]elastic.ClientOp
 		}
 	} else {
 		httpTransport := &http.Transport{
-			Proxy: http.ProxyFromEnvironment,
```

Comment:
revert all the changes in this function except line 285. It breaks a couple of things  e.g. https://github.com/jaegertracing/jaeger/commit/47d20296982d2e01432be46daef1dfc2c365872b#diff-06fc7142fac7ac3ed9db42948f2558c8R306

Reply:
yep, i've got a TODO still to restore this